ui/components/app/alert-system/alert-modal/alert-modal.test.tsx

```diff
@@ -165,7 +165,8 @@ describe('AlertModal', () => {
       fieldAlerts: [alertsMock[1]],
       getFieldAlerts: () => [],
       isAlertConfirmed: () => false,
-    });
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } as any);
```

Comment:
nit: Just to remove the `any` we could use `(useAlertsSpy as jest.Mock).mockReturnValue({`